packages/mdc-top-app-bar/README.md

```diff
@@ -0,0 +1,122 @@
+<!--docs:
+title: "Toolbars"
+layout: detail
+section: components
+excerpt: "A container for multiple rows of items such as application title, navigation menu, or tabs."
+iconId: toolbar
+path: /catalog/topappbar/
+-->
+
+# Top App Bars
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/toolbar/index.html">
```

Comment:
We might want to leave this out for the time being... Otherwise, this should presumably point to top-app-bar.html, and the next line's filename and alt text should be changed (and we'll need to provide an image for it, Bonnie knows how that process works)